ts/features/idpay/payment/screens/IDPayPaymentCodeInputScreen.tsx

```diff
@@ -53,40 +55,35 @@ const IDPayPaymentCodeInputScreen = () => {
           <VSpacer size={16} />
           <Body>{I18n.t("idpay.payment.manualInput.subtitle")}</Body>
           <VSpacer size={40} />
-          {/* FIXME replace with the new input field from the Design System 2.0  */}
-          <LabelledItem
-            isValid={isInputValid}
-            accessibilityLabel={I18n.t("idpay.payment.manualInput.input")}
-            inputMaskProps={{
-              type: "custom",
-              options: { mask: "SSSSSSSS" },
-              keyboardType: "default",
-              returnKeyType: "done",
-              value: inputState.value,
-              autoCapitalize: "none",
-              placeholder: I18n.t("idpay.payment.manualInput.input"),
-              onChangeText: value => {
-                setInputState({
+          <TextInput
+            onChangeText={value => {
+              setInputState({
+                value,
```

Comment:
```suggestion
                value: value.toUpperCase(),
```

What do you think about uppercasing the typed value? I think it might help readability.
cc: @thisisjp

Reply:
Btw, @dmnplb @CrisTofani we should add the `autoCapitalize` prop to this component to allow auto-uppercase. What do you think?

Reply:
Here we go: https://github.com/pagopa/io-app-design-system/pull/110
@forrest57

Reply:
purrrrfect! 🐈‍⬛

Reply:
adressed in [d8fcfea](https://github.com/pagopa/io-app/pull/5136/commits/d8fcfea5a0bbef5c6f1d605182873dfa417e5ab1) 👍